### ISSUE TITLE
Documentation menu is hard to read

### ISSUE THREAD

**Issue by mangelozzi** [encode/django-rest-framework#8366]
The menu for the smaller screens documentation (at https://www.django-rest-framework.org/) has such low contrast, I cant read it, I have to hover over each item one by one so that its black/white.

In Chrome it says it has a contrast ratio of 2.19, which is a fail (should be at least 4.5):
![image](https://user-images.githubusercontent.com/19764314/154219679-a1f3bc5c-5a64-481d-b9d1-94d6982cd14a.png)

Here you can see the red on the gray is:
![image](https://user-images.githubusercontent.com/19764314/154219323-170bb315-374e-4409-94a6-44b8f87c1071.png)
